content/zh/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,372 @@
+
+<!--
+## Kubernetes RBAC - privilege escalation risks {#privilege-escalation-risks}
+
+Within Kubernetes RBAC there are a number of privileges which, if granted, can allow a user or a service account
+to escalate their privileges in the cluster or affect systems outside the cluster.
+
+This section is intended to provide visibility of the areas where cluster operators 
+should take care, to ensure that they do not inadvertantly allow for more access to clusters than intended.
+-->
+## Kubernetes RBAC - 特权提权的风险 {#privilege-escalation-risks}
+
+在 Kubernetes RBAC 中有许多特权，如果被授予，
+用户或服务帐户可以提升其在群集中的权限并可能影响集群外的系统。
+
+本节旨在为集群操作员提供应注意所在区域的可见性，
+以确保它们不会无意中允许对集群的访问超出预期。
+
+<!--
+### Listing secrets
+
+It is generally clear that allowing `get` access on Secrets will allow a user to read their contents.
+It is also important to note that `list` and `watch` access also effectively allow for users to reveal the Secret contents.
+For example, when a List response is returned (for example, via `kubectl get secrets -A -o yaml`), the response
+includes the contents of all Secrets.
+-->
+### 列举 Secret {#listing-secrets}
+
+通常很清楚，若设置 Secrets 的 `get` 访问权限，
+用户可以获取其内容。 同样重要需要注意的是： `list` 和
+`watch` 访问也允许用户获取 Secret 的内容。
+例如，当返回 List 响应时（如,通过 
+`kubectl get secrets -A -o yaml`），响应包括所有秘密的内容。
+
+<!--
+### Workload creation
+
+Users who are able to create workloads (either Pods, or
+[workload resources](/docs/concepts/workloads/controllers/) that manage Pods) will
+be able to gain access to the underlying node unless restrictions based on the Kubernetes
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/) are in place.
+-->
+### 工作负载的创建 {#workload-creation}
+
+能够创建工作负载的用户（Pod 或 管理 Pod 的
+[工作负载资源](/zh/docs/concepts/workloads/controllers/)） 
+能够访问底层节点，除非基于 Kubernetes 的
+[Pod 安全标准](/zh/docs/concepts/security/pod-security-standards/)做限制。
+
+<!--
+Users who can run privileged Pods can use that access to gain node access and potentially to
+further elevate their privileges. Where you do not fully trust a user or other principal
+with the ability to create suitably secure and isolated Pods, you should enforce either the
+**Baseline** or **Restricted** Pod Security Standard.
+You can use [Pod Security admission](/docs/concepts/security/pod-security-admission/)
+or other (third party) mechanisms to implement that enforcement.
+-->
+可以运行特权 Pod 的用户可以利用该访问权限获得节点访问权限，
+并可能进一步提升他们的特权。如果你不完全信任具有创建适当安全和隔离 
+Pod 的能力的用户或其他委托人，你应该按照 **Baseline**
+或 **Restricted** Pod 安全标准强制执行。
+您可以使用 
+[Pod 安全性准入](/zh/docs/concepts/security/pod-security-admission/)
+或其他（第三方）机制来实施该执行。
+
+<!--
+You can also use the deprecated [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) mechanism
+to restrict users' abilities to create privileged Pods (N.B. PodSecurityPolicy is scheduled for removal
+in version 1.25).
+-->
+您还可以使用已弃用的 [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+机制以限制用户创建特权 Pod 的能力 （特别注意, PodSecurityPolicy 已计划在版本 1.25 中删除）。
```

Comment:
```suggestion
机制以限制用户创建特权 Pod 的能力 （特别注意：PodSecurityPolicy 已计划在版本 1.25 中删除）。
```

Reply:
nice catch, thanks。 I'm just wondering if a comma is better or a colon